src/backend/base/langflow/components/nvidia/nvidia_ingest.py

```diff
@@ -0,0 +1,218 @@
+import logging
+from pathlib import Path
+from urllib.parse import urlparse
+
+from nv_ingest_client.client import Ingestor
+from nv_ingest_client.util.file_processing.extract import EXTENSION_TO_DOCUMENT_TYPE
+
+from langflow.custom import Component
+from langflow.io import BoolInput, DropdownInput, FileInput, IntInput, Output, StrInput
+from langflow.schema import Data
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
from loguru import logger
```
@ogabrielluiz what do you suggest? or do you suggest `from langflow.logging.logger import  logger`

Reply:
loguru is correct.